readme.md

```diff
@@ -144,6 +147,12 @@ Accepts keywords which ends up as a path in Analytics.
 
 `.track('init', 'backbone')` becomes `/init/backbone`
 
+#### .trackEvent(category, action, [label, value])
```

Comment:
Any specific reason? I was trying to follow the `track` methods signature. Agree options object would be flexible than arguments